resolwe_bio/processes/expression/featureCounts_beta.py

```diff
@@ -0,0 +1,980 @@
+"""featureCounts."""
+import gzip
+import io
+import json
+import shutil
+from pathlib import Path
+
+import pandas as pd
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    FloatField,
+    GroupField,
+    IntegerField,
+    JsonField,
+    SchedulingClass,
+    StringField,
+)
+
+from resolwe_bio.process.runtime import ProcessBio
+
+
+def get_gene_counts(infile, outfile, sample_name):
+    """Rename columns and extract gene counts from featureCounts output file."""
+    exp = pd.read_csv(
+        infile,
+        sep="\t",
+        skip_blank_lines=True,
+        header=1,
+        index_col="Geneid",
+        dtype={
+            "Geneid": str,
+        },
+        squeeze=True,
+    )
+    filter_col = [col for col in exp if col.startswith(sample_name)]
+
+    if len(filter_col) > 1:
+        exp["sum_count"] = exp[filter_col].sum(axis=1)
+        exp = exp.astype({"sum_count": int})
+
+        return exp[["sum_count"]].to_csv(
+            outfile,
+            index_label="FEATURE_ID",
+            header=["EXPRESSION"],
+            sep="\t",
+        )
```

Comment:
The insides of if statment could just sum counts and specify the return column. Then the return can be the same in both cases.
```
    if len(filter_col) > 1:
        return_column = "sum_count"
        exp[return_column] = exp[filter_col].sum(axis=1)
    else:
        return_column = sample_name


    exp = exp.astype({return_column: int})
    return exp[[return_column]].to_csv(
            outfile,
            index_label="FEATURE_ID",
            header=["EXPRESSION"],
            sep="\t",
    )
```